src/components/Dropdown/styles/_styles.scss

```diff
@@ -0,0 +1,35 @@
+.dropdown__btn {
+	 @include cx-btn--size(34px);
+	 margin-bottom: 0px;
+	 position: relative;
+	 background-color: $gb-color-component-dark;
+	 color: black;
+	 border: none;
+	 	&:hover{
+	 		cursor: pointer;
+	 	}	  
+}
+.dropdown-content {
+	display: block;
+	margin-top: 0px;
+	width: 200px;
+	background-color: $gb-color-component-dark;
+	color: black;
+	text-align: left;
+		&__list {
+		margin-top: 0px;
+		margin-left: 7px;	
```

Comment:
Margins and passings should be taken from `spacing` variables - please reuse those.